pom.xml

```diff
@@ -33,11 +33,11 @@
         <dice-provider-artifact-id>dice-provider</dice-provider-artifact-id>
         <x509-provider-artifact-id>x509-provider</x509-provider-artifact-id>
 
-        <iot-device-client-version>1.28.0</iot-device-client-version>
-        <iot-service-client-version>1.26.0</iot-service-client-version>
-        <iot-deps-version>0.11.0</iot-deps-version>
-        <provisioning-device-client-version>1.8.5</provisioning-device-client-version>
-        <provisioning-service-client-version>1.7.0</provisioning-service-client-version>
+        <iot-device-client-version>1.29.0</iot-device-client-version>
+        <iot-service-client-version>1.27.0</iot-service-client-version>
+        <iot-deps-version>0.11.1</iot-deps-version>
```

Comment:
q - is "deps" here similar to the .NET shared nuget:
- all other nuget packages have a dependency on shared
- if we release "shared", we have to release all the dependent nugets as well.

Reply:
Yes. We also have lots of code refactoring in these libraries (prov-dev, prov-service) that may as well be released anyways

Reply:
won't we need to release the security provider and tpm jar as well then?